script/pgsqlms

```diff
@@ -1021,6 +1038,11 @@ sub pgsql_validate_all {
     @content = <$fh>;
     close $fh;
 
+    unless ( $maxlag =~ /^[0-9,.E]+$/ ) {
```

Comment:
As discussed on instant messaging, I am waiting for the very last commit to fix this regexp to a cleaner code using `Scalar::Util::looks_like_number()`

Thank you!